packages/lesswrong/components/ea-forum/EABestOfPage.tsx

```diff
@@ -585,7 +293,11 @@ const EABestOfPage = ({ classes }: { classes: ClassesType }) => {
   );
 };
 
-const EABestOfPageComponent = registerComponent("EABestOfPage", EABestOfPage, { styles });
+const EABestOfPageComponent = registerComponent(
+  "EABestOfPage",
+  EABestOfPage,
+  {styles, stylePriority: 2},
```

Comment:
What's the reason for the style priority here?

Reply:
There was some component we were using (I don't remember exactly which) that was taking priority over this. I'd normally be a bit hesitant to do this, but I feel like any `...Page` component should just have infinitely high style priority. If I'm defining styles at the page level they should always win.